layouts/fastly/helix.vcl

```diff
@@ -22,23 +146,29 @@ sub hlx_strain {
   if (req.http.Cookie:X-Strain) {
     set req.http.X-Strain = req.http.Cookie:X-Strain;
   }
-  # we don't need cookies for anything else
-  unset req.http.Cookie;
+
+  # Sanitize user input. `urlencode` leaves alphanumeric and `-._~`
+  set req.http.X-Strain = regsuball(urlencode(req.http.Foo), {"%.."}, "_");
```

Comment:
`req.http.foo` = `req.http.X-Strain`?

Reply:
@drwilco I think @ejthurgo is correct. should we rename this?